src/bento/components/bento-video/1.0/base-element.js

```diff
@@ -4,7 +4,21 @@ import {CSS as CSS_AUTOPLAY} from './autoplay.jss';
 import {BentoVideo} from './component';
 import {CSS} from './component.jss';
 
-export class BentoVideoBaseElement extends PreactBaseElement {}
+export class BentoVideoBaseElement extends PreactBaseElement {
+  /** @override */
+  init() {
+    const {element} = this;
+    return {
+      'onPlayStateChange': (isPlaying) => {
```

Comment:
We don't have any e2e test coverage for this standalone web-component. We'll need to address that later to make sure this is working as intended. e2e testing can sometimes be very tricky, so I don't think it should be a blocker for this PR.